src/database/models/article.js

```diff
@@ -1,33 +1,46 @@
 module.exports = (sequelize, DataTypes) => {
-  const Article = sequelize.define(
-    'Article',
-    {
-      title: DataTypes.STRING,
-      body: DataTypes.TEXT,
-      image: DataTypes.STRING,
-      published: DataTypes.BOOLEAN,
-      publisheddate: {
-        type: DataTypes.DATE,
-        field: 'published_date',
-        defaultValue: new Date()
-      },
-      status: {
-        type: DataTypes.ENUM('draft', 'active', 'deactivated'),
-        defaultValue: 'draft'
-      },
-      authorId: DataTypes.INTEGER,
-      slug: DataTypes.STRING,
-      description: DataTypes.STRING
+  const Article = sequelize.define('Article', {
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        len: [2]
+      }
     },
-    {}
-  );
+    description: DataTypes.STRING,
+    body: DataTypes.TEXT,
+    image: DataTypes.STRING,
```

Comment:
I do not see a `authotId` column. I think there should be one to reference the author

Reply:
Thank you for this feedback.
The authorId column is automatically generated and included in the Article Table through the [`belongsTo`](http://docs.sequelizejs.com/manual/associations.html#belongsto) association between the User Model (target) and the Article Model (source)
```
    Article.belongsTo(models.User, {
      through: 'Articles',
      foreignKey: 'id',
      as: 'authorId'
    });
```